test/TokenomicsLoop.t.sol

```diff
@@ -477,6 +477,8 @@ contract TokenomicsLoopTest is BaseSetup {
         // The number of services is within the max number range
         numServices = bound(numServices, 1, maxNumUnits);
 
+        vm.pauseGasMetering();
```

Comment:
This is a great cheat code that was found - somehow internal gas consumption has increased and tests started to break due to OutOfGas error. This cheat code helps fixing this issue.